specs/interop/derivation.md

```diff
@@ -132,19 +133,31 @@ is a deposit transaction sent to the zero address.
 
 This transaction MUST have the following values:
 
-1. `from` is `0xdeaddeaddeaddeaddeaddeaddeaddeaddead0002` (the address of the
+1. `from` is `0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001` (the address of the
```

Comment:
Original thought was that it would be from a unique address which makes it easy to identify this transaction without having to parse the data (and eliminates any risk that you could create a transaction that looks the same, though I think there are other factors that prevent that currently anyway). 

I'm not strongly opinionated either way but it seemed like a good idea at the time (just missed updating the part in brackets).

Reply:
Ok, let's change the address to `...0002`. I'll update the implementation too